boring/src/ssl/mod.rs

```diff
@@ -2240,6 +2240,16 @@ impl ClientHello<'_> {
     pub fn version_str(&self) -> &'static str {
         self.ssl().version_str()
     }
+
+    /// Returns the client hello raw data
+    pub fn raw_client_hello(&self) -> &[u8] {
+        unsafe { slice::from_raw_parts(self.0.client_hello, self.0.client_hello_len) }
+    }
+
+    /// Returns the client random raw data
+    pub fn raw_random(&self) -> &[u8] {
```

Comment:
I think `random` is a better name, "raw" feels weird to me as there could not be a non-raw version of this, right?

Reply:
Good point, not much you can parse, I'll rename it.